tools/make-rpms

```diff
@@ -50,22 +54,59 @@ while [ $# -gt 0 ]; do
     shift
 done
 
-if [ $# -gt 1 ]; then
+if [ $# -gt 1 -o $# = 1 -a -n "${build_in_place}" ]; then
     usage
     exit 2
 fi
 
-srpm="$($base/tools/make-srpm ${1-})"
-
 tmpdir=$(mktemp -d $PWD/rpm-build.XXXXXX)
+
+if [ -n "${build_in_place}" ]; then
+    test -f "${base}/configure" || NOCONFIGURE=1 "${base}/autogen.sh"
+    version="$(sed -n "s/^PACKAGE_VERSION='\(.*\)'\$/\1/p" "${base}/configure")"
+    test -n "${version}"
```

Comment:
Can this re-use tools/make_dist.py, so that we factorize that stuff a bit, and make use of jumpstart?

Reply:
I don't think that's appropriate.  We're not at all interested in making a tarball here, or even running `./configure` for that matter.  All we want to do is make sure `./configure` exists so that the spec file can run it.